src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/repositories/mapping/TakeMapper.kt

```diff
@@ -22,7 +22,7 @@ class TakeMapper {
     fun mapToEntity(obj: Take): TakeEntity {
         return TakeEntity(
                 obj.id,
-                null,
+                -1, // Not a valid value. Must be overridden before DB insert
```

Comment:
Is there a time when we're mapping and don't have the chunk id?

Reply:
Mapping from Take to TakeEntity, we do not have the id of the chunk. Although the Take does always belong to another chunk, that information is not stored in the Take object.

An alternative solution would be to pass the owning chunk into the mapper function as well and initialize the TakeEntity with that chunk's id. Currently, the chunk id is set in the take repository after the mapper has been called.

Reply:
I think I prefer passing in the chunk to the mapper